apps/www/components/Article/Page.js

```diff
@@ -266,6 +266,7 @@ const ArticlePage = ({
   const articleContent = article?.content
   const articleUnreadNotifications = article?.unreadNotifications
   const routerQuery = router.query
+  const isClimate = article?.content?.meta?.climate
```

Comment:
```suggestion
  const isClimate = !!article?.content?.meta?.climate
```
use '!!' to convert possible undefined to false, since the isClimate prop on the frame expects a boolean.